.gitattributes

```diff
@@ -0,0 +1 @@
+alternate_formats/* -diff
```

Comment:
Could you say what this `.gitattributes` file is doing? I haven't used it before, and I did just go and try to learn more about it, but I'm having a hard time understanding what this line does.